src/models/recipe/recipe.types.ts

```diff
@@ -58,4 +63,11 @@ export interface IRecipeModel extends Model<IRecipeDocument> {
   }: {
     userId: string;
   }) => Promise<IRecipeDocument[]>;
+  deleteRecipeById: ({
+    userId,
+    recipeId,
+  }: {
+    userId: string;
+    recipeId: string;
+  }) => Promise<IRecipeDocument[]>;
```

Comment:
Let's put this instead on the `IRecipeModel` (instead of `IRecipeDocument`)

Reply:
(res, req) still complains that it is of type 'any'